tutorial/README.md

```diff
@@ -0,0 +1,413 @@
+#Tutorial#
+
+The [Spark Data Sources API](https://databricks.com/blog/2015/01/09/spark-sql-data-sources-api-unified-data-access-for-the-spark-platform.html) introduced in Spark 1.2 supports a pluggable mechanism for integration with structured data-sources. It is a unified API designed to support two major operations:
+
+1. Loading structured data from an external data source into Spark
+2. Storing structured data from Spark into an external data source.
+
+The Data Sources API has built in integration for several data sources such as Hive, Avro, Json, JDBC and Parquet. Third party integration is added through spark-packages. One such integration is for Amazon Redshift which is provided by the spark-redshift package. 
```

Comment:
JSON is usually in all-caps, in my experience, since it's an acronym. Third-party should be hyphenated.